peak/family.py

```diff
@@ -225,6 +225,8 @@ def get_adt_t(self, adt_t):
     def get_constructor(self, adt_t):
         return adt_t
 
+    def assemble(self, locals, globals, **kwargs):
+        return super().assemble(locals, globals, **kwargs)
```

Comment:
shouldn't need this

Reply:
I was getting an error because the parent class is abstract

Reply:
Ah I seea